scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -299,8 +304,19 @@ public PublicKey getPublicKey() {
 	 * 
 	 * @return the root certificates
 	 */
+	@Deprecated
```

Comment:
I just deprecate all method, but I think we should remove it instead.

Reply:
I would remove it now.